tests/test_cli.py

```diff
@@ -63,3 +64,12 @@ def check_options_for_duplicates(options: Dict[str, List[str]]) -> Set[str]:
     return set(
         [flag for flag in flags if (flags.count(flag) > 1 or flag in RESERVED_FLAGS)]
     )
+
+
+def test_loaded_modules(runner):
+    should_not_load = {"git"}
+
+    runner.invoke(["--help"])
```

Comment:
I would execute command like "sql" because `--help` in interpreted on very early step of command processing and CLI exits and because of that we might overlook some unwanted imports.

Reply:
Changed to sql